example/image-classification/common/fit.py

```diff
@@ -168,10 +168,16 @@ def fit(args, network, data_loader, **kwargs):
 
     # callbacks that run after each batch
     batch_end_callbacks = [mx.callback.Speedometer(args.batch_size, args.disp_batches)]
+    eval_end_callbacks = []
     if 'batch_end_callback' in kwargs:
         cbs = kwargs['batch_end_callback']
         batch_end_callbacks += cbs if isinstance(cbs, list) else [cbs]
 
+    # tensorboard logs
+    if "eval_end_callback" in kwargs:
+	cbs = kwargs['eval_end_callback']
+        eval_end_callbacks += cbs if isinstance(cbs, list) else [cbs]
```

Comment:
I verified this in current master branch, but there's some problem about `common/fit.py` as below:
train_mnist.py failed: TypeError: __init__() got an unexpected keyword argument 'multi_precision' · Issue #7424 · apache/incubator-mxnet
https://github.com/apache/incubator-mxnet/issues/7424

I verified that runs in MXNet 0.10.0 Release is okay.